thread-discussions.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: 'Thread Discussions'
+---
+
+<Frame>![Example Thread Discussion](/public/images/thread-discussion.png)</Frame>
+
+Thread discussions allow you to lead side conversations for any thread with your wider team.
+
+Simply get the link to the thread in Plain and paste it in any of your internal slack channels.
```

Comment:
I would avoid 'simply', 'just' etc. If the user struggles to get this to work it then feels weird/odd
```suggestion
To start a discussion you can copy a link to a thread in Plain and share it in Slack channel.
```